emoji-copy@felipeftn/extension.js

```diff
@@ -258,26 +222,21 @@ class EmojisMenu {
 }
 
 function init() {
-    ExtensionUtils.initTranslations('emoji-copy');
     try {
-        let theme = imports.gi.Gtk.IconTheme.get_default();
-        theme.append_search_path(Me.path + '/icons');
+        let theme = Gtk.IconTheme.get_default();
+        theme.append_search_path(ExtensionUtils.path + '/icons');
```

Comment:
This should probably use `Extension.path` instead of `ExtensionUtils.path`, but tbh I'm not 100% sure if I understand [upstream's documentation](https://gjs.guide/extensions/upgrading/gnome-shell-45.html#extensionutils) correctly.